### DISCUSSION TITLE
Identifier tool

### DISCUSSION THREAD

**Discussion by Zehir** [deconz-community/node-red-contrib-deconz#170] [Ideas]
Display all devices in a grid and make them blink on event.
This will be usefull when you have 5 identical switchs and don't know who is who.
You can also send identify on light.

**Comment by TheNON75** [comment]
Sounds okay, but not the top prio in my opinion.
Phoscon can identify the lights

Frankly speaking a device map would be great too or maybe better

**Reply by Zehir** [reply]
What you mean by a device map ?
